pkg/controller/controller.go

```diff
@@ -109,6 +110,7 @@ func NewUnmanaged(name string, mgr manager.Manager, options Options) (Controller
 		MaxConcurrentReconciles: options.MaxConcurrentReconciles,
 		SetFields:               mgr.SetFields,
 		Name:                    name,
+		Log:                     log.RuntimeLog.WithName("controller").WithName(name),
```

Comment:
I think it would be better to keep the fields the same as they are today by doing
```
log.RuntimeLog.WithName("controller").WithValues("controller", name)
```

Changing this might break ppls queries

Reply:
Makes sense, although I've also changed the other log lines to explicitly call `name` and `namespace`; do we consider logs to be an API within this project?

Reply:
> Makes sense, although I've also changed the other log lines to explicitly call name and namespace;

That `name` and `namespace` is from the `reconcile.Request` and not the name of the controller or WDYM?

> do we consider logs to be an API within this project?

Probably not the same as we consider other apis but I think we should try to avoid breaking it unless we have a good reason to.

Reply:
That's fair, I don't have strong objections on the name, I've changed it because it kinda makes sense to have the name as the full path like `controller-runtime/controller/object-reconciler` instead of having it separately.

> That name and namespace is from the reconcile.Request and not the name of the controller or WDYM?
Right, what I meant is that I changed the request from being logged like `"request", req` to a more explicit `req.Name, "namespace", req.Namespace` (line 258)

Reply:
> That's fair, I don't have strong objections on the name, I've changed it because it kinda makes sense to have the name as the full path like controller-runtime/controller/object-reconciler instead of having it separately.

I disagree, filtering logs by controller is a very common usecase and that is easier if there is a dedicated field for that rather than needing to know an arcane prefix.

Reply:
Fixed